setup.py

```diff
@@ -101,6 +101,47 @@ def parse_arg_remove_string(argv, arg_name_equal):
 is_manylinux = environ.get('AUDITWHEEL_PLAT', None) in manylinux_tags
 
 
+def build_torch_cpp_extensions():
+    # Run this from build dir (e.g. (...)/build/Linux/RelWithDebInfo/)
+    is_gpu_available = False
+    try:
+        import torch
+        is_gpu_available = torch.cuda.is_available()
+    except ImportError:
+        pass
+
+    ############################################################################
+    # Pytorch CPP Extensions that DO require CUDA/ROCM
+    ############################################################################
+    if is_gpu_available:
+        ret_code = subprocess.call(
+            f"{sys.executable} torch_cpp_extensions/torch_gpu_allocator/setup.py build {'--use_rocm' if rocm_version else ''}", shell=True
+        )
+        if ret_code != 0:
+            print('There was an error compiling a PyTorch CPP extension called "torch_gpu_allocator"')
+            sys.exit(ret_code)
```

Comment:
May we create a helper method with this implementation? Seems like it is being used twice in this function.

Reply:
we have a work item to properly handle cpp extensions. we can take this on that work item